ntbs-service/DataAccess/DataQualityRepository.cs

```diff
@@ -101,7 +113,8 @@ public async Task<IList<Notification>> GetNotificationsEligibleForDataQualityTre
         private IQueryable<Notification> GetBaseNotificationQueryableForAlerts()
         {
             return _context.Notification
-                .Include(n => n.HospitalDetails);
+                .Include(n => n.HospitalDetails)
+                .Include(n => n.TreatmentEvents);
```

Comment:
Do we want to have a secondary `GetNotificationQueryableWithTreatmentEvents` for the includes? Just to prevent unnecessary loading.

Reply:
Ping